tests/test_bgp_af.rb

```diff
@@ -1081,4 +1141,98 @@ def route_target_tester(bgp_af, af, opts, should, test_id)
                    "#{test_id} : #{af} : route_target_#{opt}_evpn")
     end
   end
+
+  ##
+  ## suppress_inactive
+  ##
+  def suppress_inactive(asn, vrf, af)
+    bgp_af = RouterBgpAF.new(asn, vrf, af)
+
+    #
+    # Set and verify
+    #
+    val = false
+    bgp_af.suppress_inactive = val
+    assert_equal(val, bgp_af.suppress_inactive,
+                 'Error: suppress inactive value does not match set value')
+
+    val = true
+    bgp_af.suppress_inactive = val
+    assert_equal(val, bgp_af.suppress_inactive,
+                 'Error: suppress inactive value does not match set value')
+
+    val = bgp_af.default_suppress_inactive
+    bgp_af.suppress_inactive = val
+    assert_equal(val, bgp_af.suppress_inactive,
+                 'Error: suppress inactive value does not match default' \
+                 'value')
+  end
+
+  def test_suppress_inactive
+    afs = [%w(ipv4 unicast), %w(ipv6 unicast)]
+    afs.each do |af|
+      suppress_inactive(55, 'red', af)
+    end
+  end
+
+  ##
+  ## table_map
+  ##
+  def table_map(asn, vrf, af)
+    bgp_af = RouterBgpAF.new(asn, vrf, af)
+
+    #
+    # Set and verify
+    #
+    val = 'sjc'
+    bgp_af.table_map_set(val)
+    assert_equal(val, bgp_af.table_map,
+                 'Error: default metric value does not match set value')
+
+    val = bgp_af.default_table_map
+    bgp_af.table_map_set(val)
+    assert_equal(val, bgp_af.table_map,
+                 'Error: default metric value does not match default' \
+                 'value')
+  end
+
+  def test_table_map
+    afs = [%w(ipv4 unicast), %w(ipv6 unicast)]
+    afs.each do |af|
+      table_map(55, 'red', af)
+    end
+  end
+
+  ##
+  ## table_map_filter
+  ##
+  def table_map_filter(asn, vrf, af)
```

Comment:
Nit: This test is closely related to test_table_map. Consider moving these tests into test_table_map.